### DISCUSSION TITLE
Any plans on adding more sources?

### DISCUSSION THREAD

**Discussion by ghost** [darylhjd/mangadesk#57] [Ideas]
I have been using mangadesk the past few days, while it is incredible and really versitile and conveinent, and I don't want to sound ungrateful since I really love this amazing tool. But is there any plans in the near future to either add more sources or be able to add custom sources?

**Comment by darylhjd** [comment]
im glad mangadesk has proven to be useful for you :)

unfortunately, i have no plans to add or support more sources in the foreseeable future, as it would simply be too much work for me. 

i had originally created this app to be a substitute while the mangadex website was down. it was never meant to be more than a simple manga downloader for mangadex, and it would likely stay that way. 

of course, if other developers/i ever feel the urge to add more features, we may see such support being added, but it seems unlikely for now.

i hope you understand :>
